### PR TITLE
feat(MenuDropdown): Add DoubleCaretVertical icon and trailingIcon prop to Dropdown.Button

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Closes: https://github.com/influxdata/clockface/issues/803
### Changes

Also adds the double caret icon for menuDropdown.

Screen Shot 2022-07-18 at 1 50 04 PM

### Screenshots

// Add screenshots here if relevant

### Checklist
Check all that apply

- [ ] Updated documentation to reflect changes
- [ ] Added entry to top of Changelog with link to PR (not issue)
- [ ] Tests pass
- [ ] Peer reviewed and approved
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: src/Components/Dropdowns/DropdownButton.tsx [hunk 1]
```diff
@@ -90,6 +93,9 @@ export const DropdownButton = forwardRef<
         <span className="cf-dropdown--selected">{children}</span>
         {status === ComponentStatus.Loading ? (
           <TechnoSpinner diameterPixels={20} />
+        ) : !!trailingIcon ? (
```
Comment (wdoconnell) [inline src/Components/Dropdowns/DropdownButton.tsx:96]: I now see why you did the above though - if we set `trailingIcon` to a default, we still need to adjust the CSS style.

**Comment by wdoconnell** [comment]
Pending my two comments, this looks good to me. But we should make sure that we don't use this in the UI until the missing clockface icons are reverted.

File: src/Components/Dropdowns/DropdownButton.tsx (hunk 1 above)
Comment (ChitlangeSahas) [reply src/Components/Dropdowns/DropdownButton.tsx:96]: Yes! one of the classnames has a animation that we do not want for our vertical caret.

**Review by wdoconnell — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**